src/UIAutomation/UiaOperationAbstraction/UiaOperationAbstraction.h

```diff
@@ -1706,15 +1720,20 @@ namespace UiaOperationAbstraction
         explicit UiaVariant(UiaDouble value);
         explicit UiaVariant(UiaString value);
 
-        template <typename ComEnumT, typename WinRTEnumT, typename StandinT>
-        explicit UiaVariant(UiaEnum<ComEnumT, WinRTEnumT, StandinT> value) :
+        template <typename ComEnumT, typename WinRTEnumT, typename StandinT, CastFuncType<StandinT> CastFunc>
+        explicit UiaVariant(UiaEnum<ComEnumT, WinRTEnumT, StandinT, CastFunc> value) :
             UiaTypeBase(
                 value.IsRemoteType() ?
-                UiaVariant(static_cast<AutomationRemoteObject>(static_cast<typename UiaEnum<ComEnumT, WinRTEnumT, StandinT>::RemoteType>(value))) :
+                UiaVariant(static_cast<winrt::Microsoft::UI::UIAutomation::AutomationRemoteObject>(static_cast<typename UiaEnum<ComEnumT, WinRTEnumT, StandinT, CastFunc>::RemoteType>(value))) :

```

Comment:
Interestingly, I have to add `winrt::Microsoft::UI::UIAutomation::` before `AutomationRemoteObject` to get the new test to compile correctly, and this change totally makes sense.
But that means this constructor is not actually even compiled once before the new test 🤣